src/pages/[productSlug]/integrations/index.tsx

```diff
@@ -66,6 +73,10 @@ export async function getServerSideProps({ params }) {
 	]
 	return {
 		props: {
+			metadata: {
+				title: `Integrations | ${product.name}`,
+				// description: `TODO`,
```

Comment:
👀

Reply:
Oh!  I'm leaving this here, because I _do_ want a description here, but don't know what it should be yet.  We'll have to sort that out in the new year!